test/cmdlineTests/evm_to_wasm/output

```diff
@@ -15,37 +15,37 @@ object "object" {
         function main()
         {
             let _1 := 0
-            mstore_internal(_1, _1, _1, _1, _1)
-            mstore_internal(32, _1, _1, _1, 1)
-            eth.storageStore(_1, 32)
+            mstore_internal(0:i64, _1, _1, _1, _1)
```

Comment:
The reason why there are types sometimes and not always is because type propagation is not yet implemented in the optimizer.